sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetInteroperabilitySuite.scala

```diff
@@ -165,7 +165,7 @@ class ParquetInteroperabilitySuite extends ParquetCompatibilityTest with SharedS
               // the assumption on column stats, and also the end-to-end behavior.
 
               val hadoopConf = spark.sessionState.newHadoopConf()
-              val fs = FileSystem.get(hadoopConf)
+              val fs = new Path(tableDir.getAbsolutePath).getFileSystem(hadoopConf)
```

Comment:
Can we have this pattern at https://github.com/apache/spark/pull/29357/files#diff-2f2d1e1ffed122e56f3cb20cf7c320d6R276, too?

Reply:
@dongjoon-hyun Do you mean making it 
```
// scalastyle:off FileSystemGet
val fs = FileSystem.get(hadoopConf)
// scalastyle:on FileSystemGet
```
here?

Reply:
The method Path.getFileSystem(Configuration) is suggested: https://github.com/apache/spark/pull/29357/files#diff-2f2d1e1ffed122e56f3cb20cf7c320d6R273

Reply:
Ideally we should not change the actual code when we just want to add a style check. But this is test and I think it's safe.

Reply:
You are right, @gengliangwang ~